src/patterns-hmrc-govuk-team/videos-webinars-and-email-alerts/index.njk

```diff
@@ -80,7 +75,7 @@ If you register for a webinar, you:
     })
   }}
 
-<h3 class="govuk-heading-m">Links to learning products</h3>
+<h4 class="govuk-heading-s">Links to learning products</h4>
 
 <p class="govuk-body">
 List products in this order:
```

Comment:
The `<ul>` below, should that be an `<ol>` as you're saying should specifically be in that order.

Reply:
Fixed this. Good spot.